src/arch/simddetect.cpp

```diff
@@ -85,7 +87,7 @@ SIMDDetect SIMDDetect::detector;
 bool SIMDDetect::neon_available_ = true;
 #elif defined(HAVE_NEON)
 // If true, then Neon has been detected.
-bool SIMDDetect::neon_available_;
+bool SIMDDetect::neon_available_ = false;
```

Comment:
Why this change?

Reply:
This change makes it so that when compiling for an unknown operating system, `neon_available_` defaults to `false` instead of the build failing.  It is connected to the changed further below where now `neon_available_` is no longer set when compiling for an unknown operating system.

Reply:
Without your change it is also initialized to `false`.

Reply:
Yes.  This initialisation is overridden in the constructor if running on a platform on which we know how to detect the availability of NEON.